internal/api/design/ingest.go

```diff
@@ -214,6 +214,8 @@ var _ = Service("ingest", func() {
 			// direct access to the HTTP request body reader.
 			SkipRequestBodyEncodeDecode()
 
+			Response(StatusCreated)
```

Comment:
I'd use `202 Accepted` instead of `201 Created`, there is a lot to happen until the request is completed and the resources created. Even if we are not able yet to return an identifier or URL to track the status, we should be able to do so after #1209 and #1210 are completed.

Reply:
Yeah, fair enough.  I debated 201 vs. 202 but I think you're right that 202 Accepted is more appropriate right now.